pkg/pool-utils/contracts/BasePoolFactory.sol

```diff
@@ -31,7 +31,7 @@ abstract contract BasePoolFactory is IBasePoolFactory, SingletonAuthentication,
     bool private _disabled;
 
     // Store the creationCode of the contract to be deployed by create3.
-    bytes private _creationCode;
+    bytes internal _creationCode;
```

Comment:
I don't like changing real code to accommodate tests. If it were a simple type, it could be immutable, but you can't have immutable `bytes`. Maybe we need to duplicate this in the mock?

Reply:
I guess I won't check the _creationCode.. Calling a PoolMock function using the created pool contract is enough to make sure the creation code is correct.